quarto/src/Api/GameApi.php

```diff
@@ -11,7 +11,7 @@ public function __construct() {
 
```

Comment:
Do you really need that `constructor` ?

(Otherwise, I think there's a way to decalarate functions in a namespace : [Importing and aliasing PHP functions](https://www.simonholywell.com/post/2016/10/importing-and-aliasing-php-functions/))

Reply:
Constructor isn't so useful but I didn't found how to add direct fonctions in php files
I will look your link for help
So Thanks